packages/core/data-transfer/lib/providers/local-file-destination-provider.ts

```diff
@@ -38,16 +38,26 @@ class LocalFileDestinationProvider implements IDestinationProvider {
   name: string = 'destination::local-file';
   type: ProviderType = 'destination';
   options: ILocalFileDestinationProviderOptions;
+  #providersMetadata: { source?: IMetadata; destination?: IMetadata } = {};
 
   constructor(options: ILocalFileDestinationProviderOptions) {
     this.options = options;
   }
 
-  #getDataTransformers() {
+  setMetadata(target: ProviderType, metadata: IMetadata): IDestinationProvider {
```

Comment:
I think part of my confusion might be the name here. We're not setting the destination's metadata, we're just saving the source metadata to write on close, correct? And is there any case where target would be something other than `'source'`? If so, target could be removed and we could call this something like "setSourceMetadata"

Reply:
I just can't think of a reason destination would be used for anything. But I'm not opposed to keeping it like this if you think it could be useful, as long as we have a good comment explaining it and what's going on.